src/components/DefaultStyleListDialog.vue

```diff
@@ -163,17 +163,31 @@ watch([() => props.modelValue], async ([newValue]) => {
       .filter(
         (characterInfo) => characterInfo.metas.styles.length > 1
       ) as CharacterInfo[];
-    selectedStyleIndexes.value = Object.fromEntries([
-      ...store.state.userCharacterOrder.map((speakerUuid) => [speakerUuid, 0]),
-      ...store.state.defaultStyleIds.map((defaultStyle) => [
-        defaultStyle.speakerUuid,
-        characterInfosMap.value[
-          defaultStyle.speakerUuid
-        ]?.metas.styles.findIndex(
-          (style) => style.styleId === defaultStyle.defaultStyleId
-        ) ?? 0,
-      ]),
-    ]);
+    // FIXME: エンジン未起動状態でデフォルトスタイル選択ダイアログを開くと
+    // 未起動エンジンのキャラのデフォルトスタイルが消えてしまう
+    selectedStyleIndexes.value = Object.fromEntries(
+      [
+        ...store.state.userCharacterOrder.map(
+          (speakerUuid) => [speakerUuid, 0] as const
+        ),
+        ...store.state.defaultStyleIds.map(
+          (defaultStyle) =>
+            [
+              defaultStyle.speakerUuid,
+              characterInfosMap.value[
+                defaultStyle.speakerUuid
+              ]?.metas.styles.findIndex(
+                (style) => style.styleId === defaultStyle.defaultStyleId
+              ),
+            ] as const
+        ),
+      ].filter(([speakerUuid, styleIndex]) => {
+        return (
+          characterInfosMap.value[speakerUuid]?.metas.styles[styleIndex] !==
+          undefined
+        );
+      })
```

Comment:
意図としてはこっちのほうがわかりやすいかなと思いました！
こっちでも正しく動いたので、こちらにしようかなと思います 🙇‍♂️ 
```suggestion
      ].filter(([speakerUuid]) => speakerUuid in characterInfosMap.value)
```